docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -347,95 +347,87 @@ Physical nodes (or cloud instances) are installed with a standard Centos 7 serve
-    sudo chown -R prometheus:prometheus /var/run/prometheus
-    sudo chmod +r /opt/prometheus/node_exporter-0.13.0.linux-amd64.tar
-    sudo su - prometheus (user session is now as user “prometheus”)
-    ```
+   ```sh
+   sudo mkdir /opt/prometheus
+   sudo mkdir /etc/prometheus
+   sudo mkdir /var/log/prometheus
+   sudo mkdir /var/run/prometheus
+   sudo mv /tmp/node_exporter-1.3.1.linux-amd64.tar  /opt/prometheus
+   sudo adduser prometheus # (also adds group “prometheus”)
+   sudo chown -R prometheus:prometheus /opt/prometheus
+   sudo chown -R prometheus:prometheus /etc/prometheus
+   sudo chown -R prometheus:prometheus /var/log/prometheus
+   sudo chown -R prometheus:prometheus /var/run/prometheus
+   sudo chmod +r /opt/prometheus/node_exporter-1.3.1.linux-amd64.tar
+   sudo su - prometheus (user session is now as user “prometheus”)
+   ```
 
 1. Run the following commands as user `prometheus`:
 
-    ```sh
-    cd /opt/prometheus
-    tar zxf node_exporter-0.13.0.linux-amd64.tar.gz
-    exit   # (exit from prometheus user back to previous user)
-    ```
-
-1. Edit the following file (sudo required):
-
-    ```sh
-    sudo vi /etc/systemd/system/node_exporter.service
-    ```
-
-    \
-    Add the following to `/etc/systemd/system/node_exporter.service`:
-
-      ```conf
-      [Unit]
-      Description=node_exporter - Exporter for machine metrics.
-      Documentation=https://github.com/William-Yeh/ansible-prometheus
-      After=network.target
-
-      [Install]
-      WantedBy=multi-user.target
-
-      [Service]
-      Type=simple
+   ```sh
+   cd /opt/prometheus
+   tar zxf node_exporter-1.3.1.linux-amd64.tar.gz
+   exit   # (exit from prometheus user back to previous user)
+   ```
 
-      #ExecStartPre=/bin/sh -c  " mkdir -p '/var/run/prometheus' '/var/log/prometheus' "
-      #ExecStartPre=/bin/sh -c  " chown -R prometheus '/var/run/prometheus' '/var/log/prometheus' "
-      #PIDFile=/var/run/prometheus/node_exporter.pid
+1. Edit the following file:
 
-      User=prometheus
-      Group=prometheus
+   ```sh
+   sudo vi /etc/systemd/system/node_exporter.service
+   ```
 
-      ExecStart=/opt/prometheus/node_exporter-0.13.0.linux-amd64/node_exporter  --web.listen-address=:9300 --collector.textfile.directory=/tmp/yugabyte/metrics
-      ```
 
-1. Exit from vi, and continue, as follows (sudo required):
+   Add the following to the `/etc/systemd/system/node_exporter.service` file:
+
+     ```conf
+     [Unit]
+     Description=node_exporter - Exporter for machine metrics.
+     Documentation=https://github.com/William-Yeh/ansible-prometheus
+     After=network.target
+   
+     [Install]
+     WantedBy=multi-user.target
+   
+     [Service]
+     Type=simple
+   
+     #ExecStartPre=/bin/sh -c  " mkdir -p '/var/run/prometheus' '/var/log/prometheus' "
+     #ExecStartPre=/bin/sh -c  " chown -R prometheus '/var/run/prometheus' '/var/log/prometheus' "
+     #PIDFile=/var/run/prometheus/node_exporter.pid
```

Comment:
These lines are commented out and will have no effect. They can probably be removed.